larq_compute_engine/tests/end2end_test.py

```diff
@@ -38,7 +38,9 @@ def dummy(x):
                 use_bias=False,
                 activation=activation,
             )(x)
-            x = tf.keras.layers.BatchNormalization(momentum=0.7)(x)
+            x = tf.keras.layers.BatchNormalization(
+                momentum=0.7, beta_initializer="random_normal"
```

Comment:
Using a random bias works around a known bug in the experimental_default_int8_range pass. This also allows us to re-enable a skipped test below.

Reply:
Is it known what the bug is? Can we add a comment to this file to explain why we have the random bias?